cloudify/tests/mocks/mock_rest_client.py

```diff
@@ -88,3 +93,19 @@ class MockManagerClient(object):
 
     def get_context(self):
         return {'context': {}}
+
+
+class MockAgentsClient(object):
+
+    def update(self, name, state):
+        return None
+
+    def get(self, name):
+        return Agent({
+            'id': name,
+            'name': name,
+            'state': 'started'
+        })
+
+    def create(self, name, state, node_instance_id, **kwargs):
+        return None
```

Comment:
i guess create and update also always return an Agent (or an exception), not None :P